lib/virt_feature_test_base.pm

```diff
@@ -157,7 +160,8 @@ sub post_fail_hook {
     my ($self) = shift;
 
     $self->{"stop_run"} = time();
-    assert_script_run("history -w /root/commands_history");
+    reconnect_when_ssh_console_broken unless defined(script_run("history -w /root/commands_history", die_on_timeout => 0));
```

Comment:
I am afraid this will delay our test run significantly, because test run will spin around some checkings for a long time. I think you need keep your "healthy check" simple and do not extend unintentionally, that is not necessary. I think we already over reacted to what happened. IMHO.

Reply:
'health check' and ssh reconnection are different issue.

ssh reconnection aims to handle broken ssh connection which can caused by hung supportconfg processing or host crash. fg. 
[hung supportconfg processing](https://openqa.suse.de/tests/9824014#step/hotplugging_guest_preparation/3)  //host not reboot, ssh disconnect for some reason, and our test abort and no logging uploaded in this case withoutht my pr. With my PR, the logs can be uploaded finally, [ssh reconnected](https://openqa.suse.de/tests/9831759#step/hotplugging_guest_preparation/4)

[host crash](http://10.67.129.102/tests/426#step/vgpu/113)   //host crash and reboot in this case

Reply:
Adopt ```check_port_state``` is much better and alleviate the problem that waste of time on waiting for something uncertain.